### PR TITLE
[stable10] UI test to disable user

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
UI test for disable user.
Same as: https://github.com/owncloud/user_management/pull/25

## Motivation and Context
This test tests if the user disable feature works as intended.

## How Has This Been Tested?
Tested locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by paurakhsharma** [comment]
cc. @phil-davis Please review this test

File: tests/acceptance/features/bootstrap/WebUIUsersContext.php [hunk 1]
```diff
@@ -174,9 +183,11 @@ public function theGroupNamedShouldNotBeListedOnTheWebUI($name) {
 	 * expects a table of groups with the heading "groupname"
 	 *
 	 * @param string $shouldOrNot (not|)
+	 *
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIUsersContext.php]: ``phpcs`` should complain about this extra line - param tags should be grouped together

File: tests/acceptance/features/bootstrap/WebUIUsersContext.php [hunk 2]
```diff
@@ -204,13 +215,49 @@ public function theUserReloadsTheUsersPage() {
...
+		$disabledPage->waitTillPageIsLoaded($this->getSession());
+	}
+	
+	
+
 	/**
 	 * @Then the quota of user :username should be set to :quota on the webUI
 	 *
 	 * @param string $username
+	 *
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIUsersContext.php]: Same ``phpcs`` problem here.

File: tests/acceptance/features/bootstrap/WebUIUsersContext.php [hunk 3]
```diff
@@ -223,6 +270,7 @@ public function quotaOfUserShouldBeSetToOnTheWebUI($username, $quota) {
 		}
 	}
 
+
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIUsersContext.php]: and extra blank line here

**Review by phil-davis — APPROVED** [review]
Just small formatting

**Pull request merged**